### DISCUSSION TITLE
GitHub Actions/CI FreeBSD Support

### DISCUSSION THREAD

**Discussion by cederom** [community/community#67529] [Actions] [answered]
### Select Topic Area

Product Feedback

### Body

Hello world :-)

For a long long time there is no support for FreeBSD (and others BSD Unix OS like OpenBSD, NetBSD, etc). This prevents development, testing, verification, release packages build, etc.

FreeBSD is self contained distribution of Unix descendant OS with kernel and source/binary packages included. Most of the system configuration is located in `/etc/rc.conf` and `/usr/local/etc/rc.conf`. It is stable, versatile, and older than Linux.

https://www.freebsd.org/

There are ISO and VM images available to download.

Please add FreeBSD to the Actions / CI infrastructure at GitHub.

Thank you :-)
Tomek

**Comment by jbeich** [comment, marked as answer]
Duplicate of https://github.com/actions/runner/issues/385 ?

**Reply by cederom** [reply]
True! Thanks @jbeich :-)
